pyproject.toml

```diff
@@ -1,8 +1,14 @@
+[build-system]
+requires = ["hatchling>=1.13.0"]
+build-backend = "hatchling.build"
+
 [project]
 name = "func_adl_type_generator"
 version = "0.1.0"
 description = "Commands for working with func_adl types"
+readme = "README.md"
 license = { text = "MIT" }
+requires-python = ">=3.8"
```

Comment:
I'm setting this here as it is really important to set _some_ Python version. If this needs to get bumped down to Python 3.7 that's fine, but it shouldn't be absent.